app/routes.js

```diff
@@ -225,6 +225,8 @@ module.exports.bind = function (app) {
 
   // YOUR PSP
   app.get(yourPsp.index, xraySegmentCls, permission('gateway-credentials:read'), getAccount, paymentMethodIsCard, yourPspController.getIndex)
+  app.get(yourPsp.flex, xraySegmentCls, permission('gateway-credentials:update'), getAccount, paymentMethodIsCard, yourPspController.getFlex)
```

Comment:
does this need to have update creds?

Reply:
Well I guess its like, why show someone a page if they cant change it?
but yeah technically we could just protect the post route…
but that might be frustrating to the user